src/status_im/ui/screens/desktop/main/add_new/views.cljs

```diff
@@ -70,4 +75,16 @@
        [react/touchable-highlight {:on-press #(when-not topic-error (re-frame/dispatch [:create-new-public-chat topic]))}
         [react/view {:style (merge styles/add-contact-button {:background-color (if topic-error "#eef2f5" "#4360df") })}
 
-         [react/text {:style (merge styles/add-contact-button-text {:color (if topic-error "#939ba1" :white)})} "Join public chat"]]]]]]))
+         [react/text {:style (merge styles/add-contact-button-text {:color (if topic-error "#939ba1" :white)})} "Join public chat"]]]]
```

Comment:
`topic-error` here and above can be passed as an argument to style to make all color decision in styles.